go-controller/pkg/node/node.go

```diff
@@ -367,12 +367,18 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	}
 	klog.Infof("Node %s ready for ovn initialization with subnet %s", n.name, util.JoinIPNets(subnets, ","))
 
-	// Create CNI Server
-	if config.OvnKubeNode.Mode != types.NodeModeSmartNIC {
+	if config.OvnKubeNode.Mode != types.NodeModeSmartNICHost {
```

Comment:
diff is a bit confusing.

i moved if statement from L#387 before creation of CNI Server object and set `isOvnUpEnabled`